### PR TITLE
Include Howler.js

### PR DESCRIPTION
**Pull request by chidindu-ogbonna**
This library is useful for handling audio on the web.

### CODE REVIEW

**Review by gokaygurcan — CHANGES_REQUESTED** [review]
Add new links at the end, not in the middle please.

**Comment by chidindu-ogbonna** [comment]
> Add new links at the end, not in the middle please.

Okay, done.

**Review by gokaygurcan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -875,6 +875,7 @@ Besides libraries, there're [Collection on Codepen](http://codepen.io/collection
  * [exifr](https://github.com/MikeKovarik/exifr) - The fastest and most versatile EXIF reading library. https://mutiny.cz/exifr/
  * [ts-audio](https://github.com/EvandroLG/ts-audio) - an agnostic and easy-to-use library to work with the `AudioContext` API
  * [AmplitudeJS](https://521dimensions.com/open-source/amplitudejs) - Open Source HTML5 Web Audio Library. Design your web audio player, the way you want. No dependencies required.
+  * [Howler.js](https://github.com/goldfire/howler.js) - Javascript audio library for the modern web. This makes working with audio in JavaScript easy and reliable across all platforms.
```
Comment (gokaygurcan) [inline README.md:878]: you have an additional space in front

**Comment by gokaygurcan** [comment]
If you also want to fix this, I'll wait, otherwise it doesn't break anything and we can also merge it like as is.

**Pull request closed without merging**